### ISSUE TITLE
Implement BatchNorm2D class

### ISSUE THREAD

**Issue by enrico174** [xamarin/TorchSharp#188]
Hello, Thank you very much for your great work on this project!
I am currenty working on neural network using your package, but I miss the BatchNorm2D layer.
Will it be implemented in a next release, and if yes when?
If no and if you are interested, I can contribute on this one.
Let me know, have an excellent day!

**Comment by dsyme** [comment]
@enrico174 Please do contribute

The focus for me in this project is a the level of LibTorch functions, not layers, because that is what is consumed by DiffSharp.

There are some layer wrappers implemented, as you can see, you should be able to follow Linear and other similar layers to work out how to do it and test it

**Comment by enrico174** [comment]
@dsyme Thank you very much for your reply

I understand your priorities. In deed I have seen the C++ wrappers parts, and the way they are coded seems repeatable for new layers coding on my side. I will try to see what I can do, possibly next month.

Thank you again for this project!

**Comment by GeorgeS2019** [comment]
Reference to previous issue on [BatchNorm](https://github.com/xamarin/TorchSharp/issues/112)
